libexec/config-bootstrap-fixup

```diff
@@ -12,4 +12,8 @@ if [ -z "$MIRROR_HOST" ] || [ "$MIRROR_HOST" == "127.0.0.1" ]; then
   MIRROR_HOST=$GITIAN_HOST_IP
 fi
 
-sed "s;HOSTIP;$MIRROR_HOST;g" < target-bin/bootstrap-fixup.in > target-bin/bootstrap-fixup
+if [ $APT_CACHER = "1" ]; then
+    sed "s;HOSTIP;$MIRROR_HOST;g" < target-bin/bootstrap-fixup.in > target-bin/bootstrap-fixup
+else
+    sed "s;HOSTIP:3142/;;g" < target-bin/bootstrap-fixup.in > target-bin/bootstrap-fixup
+fi
```

Comment:
This is actually not working.

`config-boostrap-fixup` is called with `bin/make-base-vm`, which have `APT_CACHER` variable through `--disable-apt-cacher`.
It is called also with `bin/gbuild`, who use it with `make-clean-vm`. At this moment, they do not have `APT_CACHER` variable anymore.

Creating the following error :
```
bash: [: =: unary operator expected
```
`APT_CACHER` will be replaced by an unquoted empty value.